### PR TITLE
LPS-132737 Isolate Product Menu, Control Menu, Add Panel, Simulation Menu, Page Audit, Page Editor Toolbar CSS

### PR DESCRIPTION
**Pull request by izaera**
Fixed CI of https://github.com/liferay-echo/liferay-portal/pull/5156

### CODE REVIEW

**Review by nhpatt — APPROVED** [review]

File: modules/apps/portal-url-builder/portal-url-builder-api/src/main/java/com/liferay/portal/url/builder/AbsolutePortalURLBuilder.java [hunk 1]
```diff
@@ -68,6 +68,12 @@
 	 * web context path.
 	 *
 	 * <p>
+	 * If the requested module resource is a JavaScript file or a stylesheet,
+	 * better use {@link #forModuleScript(Bundle, String)} or {@link
+	 * #forModuleStylesheet(Bundle, String)} instead.
+	 * </p>
+	 *
```
Comment (nhpatt) [inline modules/apps/portal-url-builder/portal-url-builder-api/src/main/java/com/liferay/portal/url/builder/AbsolutePortalURLBuilder.java:75]: Can you add @review. Brian let's you add javadocs freely if they know that they are going to be reviewed in the future. I don't know about the <p> tags though

**Comment by nhpatt** [comment]
LGTM (just a comment about javadocs) but JS conflict :(

**Comment by pat270** [comment]
@izaera I fixed conflicts, but I don't have push rights to this repo.

**Comment by pat270** [comment]
I don't know if this is related to this pr, but I'm unable to access other languages using the url `http://localhost:8080/ar/web/guest/home`. I can change the default language through Control Panel > Instance Settings > Localization, but it only applies when I'm signed out. When I'm signed in as admin, it defaults to english.

**Comment by pat270** [comment]
I was also running into caching issues. My local server wasn't updating the version of cadmin after deploying.

**Comment by Tim-Cao** [comment]
> I don't know if this is related to this pr, but I'm unable to access other languages using the url `http://localhost:8080/ar/web/guest/home`. I can change the default language through Control Panel > Instance Settings > Localization, but it only applies when I'm signed out. When I'm signed in as admin, it defaults to english.

The locale issue in URL has been fixed by https://issues.liferay.com/browse/LPS-135619

**Comment by pat270** [comment]
Thanks @Tim-Cao, I'll try it again.

**Comment by izaera** [comment]
@pat270 I gave you access :+1: . Next time, if you want/need to save time simple download the PR and resend it, as I did with the one from echo. Then we cross link them and close this.

File: modules/apps/portal-url-builder/portal-url-builder-api/src/main/java/com/liferay/portal/url/builder/AbsolutePortalURLBuilder.java (hunk 1 above)
Comment (izaera) [reply modules/apps/portal-url-builder/portal-url-builder-api/src/main/java/com/liferay/portal/url/builder/AbsolutePortalURLBuilder.java:75]: Done :+1:

**Comment by izaera** [comment]
Resent as https://github.com/liferay-frontend/liferay-portal/pull/1259

**Pull request closed without merging**